ts/src/renderer/phaser/classes/PhaserRegion.ts

```diff
@@ -13,11 +13,7 @@ class PhaserRegion extends Phaser.GameObjects.Graphics {
 
 		scene.add.existing(this);
 
-		this.transformListener = region.on('transform', () => {
-
-			this.transform();
-
-		});
+		this.transformListener = region.on('transform', this.transform, this, false); // hack for now
 
 		this.destroyListener = region.on('destroy', () => {
```

Comment:
I'm considering going with composition over inheritance so we can reuse logic such as event listeners handling even if rendering objects are not containers...

Reply:
would also fix the issue with overlapping method names